api/pb/dart/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
When this is automated, we should just copy the license file from root instead of having it in the repo twice.